sdcm/nemesis.py

```diff
@@ -4205,7 +4205,12 @@ def disrupt_add_remove_mv(self):
         Verify the MV can be used in a query.
         Finally, drop the MV.
         """
-        node1, node2 = self.cluster.nodes[:2]
+        try:
+            node1, node2 = [node for node in self.cluster.nodes if not node.running_nemesis][:2]
+        except ValueError:
+            raise UnsupportedNemesis("Not enough free nodes for nemesis. Skipping.") from ValueError
+        self.set_current_running_nemesis(node1)
+        self.set_current_running_nemesis(node2)
```

Comment:
why are you unsetting running nemesis on `node1` and `node2` ? if there is a nemesis running on them, why are you unsetting?

Reply:
i am going to set the nemesis on them, so other parallel running nemesis could be executed on them